tslint.json

```
@@ -89,9 +88,9 @@
       "check-type"
     ],
 
-    "use-input-property-decorator": true,
-    "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "no-inputs-metadata-property": true,
+    "no-outputs-metadata-property": true,
+    "no-host-metadata-property": true,
     "no-input-rename": true,
     "no-output-rename": true,
     "use-life-cycle-interface": true,
```

`use-life-cycle-interface` is renamed to `use-lifecycle-interface` ([CI log warn: `Could not find implementations for the following rules specified in the configuration: use-life-cycle-interface`](https://travis-ci.org/gund/ng-dynamic-component/jobs/552542056))

```suggestion
    "use-lifecycle-interface": true,
```